tooling/tslint-rules/noReturnAnyRule.ts

```diff
@@ -1,111 +0,0 @@
-/**
```

Comment:
This was a pretty good rule, was there a ready-made one from eslint?

Reply:
In iOS `Core` code we replaced it with [no-unsafe-return](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unsafe-return.md) rule, it should work the same